### PR TITLE
minor visual fixes

### PR DESCRIPTION
**Pull request by kangarang**
add setTxPending to remaining routes
- rm extraneous code/images
- better metamask image

rm field from create proposal form and edit copy

add mainnet modal warning component

### CODE REVIEW

File: client/pages/proposals/create.tsx [hunk 1]
```diff
@@ -55,8 +54,10 @@ const CreateProposal: React.FunctionComponent<IProps> = ({ router }) => {
       <CenteredWrapper>
         <ProposalForm onHandleSubmit={handleSubmit} />
       </CenteredWrapper>
-    </div>
+
+      <PendingTransaction isOpen={txPending} setOpen={setTxPending} />
```
Comment (akuanti) [inline client/pages/proposals/create.tsx]: I think it's probably fine to use this here for now, but this isn't really a transaction.

File: client/components/MainnetModal.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import Modal, { ModalTitle, ModalDescription } from './Modal';
+
+import React from 'react';
+import Button from './Button';
+
+const MainnetModal = ({ setMainnetModalOpen, modalIsOpen }) => {
```
Comment (akuanti) [inline client/components/MainnetModal.tsx]: I'd like to see a Storybook story for this so I can preview it.

File: client/components/NotificationsProvider.tsx [hunk 3]
```diff
@@ -12,29 +12,29 @@ export const NotificationsContext: React.Context<INotificationsContext> = React.
 export default function NotificationsProvider(props: any) {
   const [notifications, setNotifications] = React.useState<INotification[]>([]);
   const { account } = React.useContext(EthereumContext);
-  const { proposalsByID, slatesByID } = React.useContext(MainContext);
+  const { proposalsByID, slates } = React.useContext(MainContext);
```
Comment (akuanti) [inline client/components/NotificationsProvider.tsx]: What's your reasoning for changing `slatesByID` to `slates`? I don't see any changes relating to consuming that input other than in the effect.

File: client/components/NotificationsProvider.tsx [hunk 4]
```diff
@@ -12,29 +12,29 @@ export const NotificationsContext: React.Context<INotificationsContext> = React.
...
+    const result: any[] = await getNotificationsByAddress(account);
     // normalize api return data
     const normalized: INotification[] = normalizeNotifications(result, proposalsByID);
     console.info('normalized notifications:', normalized);
     // set state
     setNotifications(normalized);
   }
 
-  // runs whenever account changes
+  // runs whenever account changes or
```
[the remark this answers is not in the document]
Comment (kangarang) [reply client/components/NotificationsProvider.tsx]: haha yeah, i was debating whether to split this effect up. i settled on - it's fine for now

File: client/pages/proposals/create.tsx (hunk 1 above)
Comment (kangarang) [reply client/pages/proposals/create.tsx]: you're right. i'll make the adjustment

File: client/components/NotificationsProvider.tsx (hunk 3 above)
Comment (kangarang) [reply client/components/NotificationsProvider.tsx]: oops hah! no reasoning. my mind was thinking about how it'd look with the latest state architecture design

File: client/components/MainnetModal.tsx (hunk 2 above)
Comment (kangarang) [reply client/components/MainnetModal.tsx]: of course, please see it here: d26b48d6238d172e64492e65b4884f5dff97e5f2

File: client/pages/slates/create/grant.tsx [hunk 5]
```diff
@@ -606,6 +593,8 @@ const CreateGrantSlate: StatelessPage<IProps> = ({ query, classes, router }) =>
                   {'Create Slate'}
                 </Button>
               </Flex>
+              <Loader isOpen={txPending} setOpen={setTxPending} />
```
Comment (akuanti) [inline client/pages/slates/create/grant.tsx]: Will these both be displayed at the same time?

File: client/pages/slates/index.tsx [hunk 6]
```diff
@@ -42,6 +47,18 @@ const Slates: React.SFC = () => {
     }
   }, [slates, visibilityFilter, currentBallot.epochNumber]);
 
+  React.useEffect(() => {
```
Comment (akuanti) [inline client/pages/slates/index.tsx]: Will the modal still show up if the user navigates directly to another page?

File: client/pages/slates/create/grant.tsx (hunk 5 above)
Comment (kangarang) [reply client/pages/slates/create/grant.tsx]: woops, meant to delete this

File: client/pages/slates/index.tsx (hunk 6 above)
Comment (kangarang) [reply client/pages/slates/index.tsx]: i've adjusted it to show on all pages, but we've got another issue which is that if the user refreshes the browser, it pops up again. maybe i'll put it in local session storage

Comment (kangarang) [reply client/pages/slates/index.tsx]: done. you can test by opening up a new tab or window, which starts a new "session" of storage

**Pull request merged**